lessons/0_workflow/workflow.html

```diff
@@ -0,0 +1,120 @@
+<html>
```

Comment:
I think this would be better as just a markdown file. The markdown files are served as html pages in the github pages site of this repo ex. https://mrtim.github.io/js-munich-2020-spring/homework/1_base_page

Reply:
Slides served as normal pages, so it's really doesn't matter. 
Like here https://mrtim.github.io/js-munich-2020-spring/lessons/2_git_time/homework#/3 
We can share it in the same way. But if everyone is up for just markdown I'm fine with it as well :)

Reply:
What I don't like about https://mrtim.github.io/js-munich-2020-spring/lessons/2_git_time/homework#/3 is it is slides and I have to use arrow key to progress the slide instead of just having a page I can refer to.

Reply:
I see.
But from the other hand I know that people can't really handle long text as well :)
I personally don't care much, we just need to agree on ONE approach and stick to it. So, do we want to try markdown homework files?